### DISCUSSION TITLE
Assistance with Skyvern Captcha Handling During 2FA Login Process

### DISCUSSION THREAD

**Discussion by 4k45hv3rm4** [Skyvern-AI/skyvern#1088] [Q&A]
Hi team,

I'm using Skyvern for one of my projects, where the objective is to automate the sign-in process to a third-party website that has two-factor authentication (2FA) enabled. As part of the 2FA setup, an OTP is sent to my email, which I then retrieve using a TOTP endpoint and input to complete the login.

The issue I’m facing is with a "Verify you are human" captcha checkbox that appears on the website’s login screen (screenshot attached for reference). Despite prompting Skyvern to click on this checkbox, it seems to skip this step, moving directly to the "Continue" button without interacting with the captcha. This causes the process to get stuck and prevents successful login.

Could you please provide guidance on how to ensure that Skyvern properly interacts with the captcha checkbox before proceeding? Any advice on handling this specific type of interaction would be greatly appreciated.

Thank you!

 <img width="178" alt="Screenshot 2024-10-30 at 5 57 16 PM" src="https://github.com/user-attachments/assets/abb947eb-554b-450f-a904-d19dcaf2c717">

**Comment by suchintan** [comment]
Are you using Skyvern cloud or Skyvern OSS?

**Reply by 4k45hv3rm4** [reply]
OSS
